azurerm/internal/services/network/vpn_server_configuration_resource.go

```diff
@@ -237,14 +237,43 @@ func resourceArmVPNServerConfiguration() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"address": {
-							Type:     schema.TypeString,
-							Required: true,
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
 						},
 
 						"secret": {
-							Type:      schema.TypeString,
-							Required:  true,
-							Sensitive: true,
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+							Sensitive:    true,
+						},
```

Comment:
Your update to the docs suggests these are being replace/deprecated, is this the case?

Reply:
Azure service team said  (address, secret) can still be used to config single Radius Server. These two will still be in specification for backward compatibility. Along side, new "RadiusServers" field has been introduced into the Spec, which can
cover both single and multiple Radius Server configuration setting.  So The two fields are not officially deprecated but will, I assume, be deprecated in future.